src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts

```diff
@@ -0,0 +1,174 @@
+	}
+	run(accessor: ServicesAccessor): void {
+		accessor.get(IAccessibleViewService).previous();
+	}
+}
+registerAction2(AccessibleViewPreviousAction);
+
+
+class AccessibleViewGoToSymbolAction extends Action2 {
+	constructor() {
+		super({
+			id: 'editor.action.accessibleViewGoToSymbol',
+			precondition: accessibleViewIsShown,
+			keybinding: {
+				primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyO,
+				weight: KeybindingWeight.WorkbenchContrib + 10
+			},
+			icon: Codicon.symbolField,
+			menu: [commandPalette, accessibleViewMenu],
+			title: localize('editor.action.accessibleViewGoToSymbol', "Go To Symbol in Accessible View")
+		});
+	}
+	run(accessor: ServicesAccessor): void {
+		accessor.get(IAccessibleViewService).goToSymbol();
+	}
+}
+registerAction2(AccessibleViewGoToSymbolAction);
+
+function registerCommand<T extends Command>(command: T): T {
+	command.register();
+	return command;
+}
+
+export const AccessibilityHelpAction = registerCommand(new MultiCommand({
+	id: 'editor.action.accessibilityHelp',
+	precondition: undefined,
+	kbOpts: {
+		primary: KeyMod.Alt | KeyCode.F1,
+		weight: KeybindingWeight.WorkbenchContrib,
+		linux: {
+			primary: KeyMod.Alt | KeyMod.Shift | KeyCode.F1,
+			secondary: [KeyMod.Alt | KeyCode.F1]
+		}
+	},
+	menuOpts: [{
+		menuId: MenuId.CommandPalette,
+		group: '',
+		title: localize('editor.action.accessibilityHelp', "Open Accessibility Help"),
+		order: 1
+	}],
+}));
+
+
+export const AccessibleViewAction = registerCommand(new MultiCommand({
+	id: 'editor.action.accessibleView',
+	precondition: undefined,
+	kbOpts: {
+		primary: KeyMod.Alt | KeyCode.F2,
+		weight: KeybindingWeight.WorkbenchContrib,
+		linux: {
+			primary: KeyMod.Alt | KeyMod.Shift | KeyCode.F2,
+			secondary: [KeyMod.Alt | KeyCode.F2]
+		}
+	},
+	menuOpts: [{
+		menuId: MenuId.CommandPalette,
+		group: '',
+		title: localize('editor.action.accessibleView', "Open Accessible View"),
+		order: 1
+	}],
+}));
+
+class AccessibleViewDisableHintAction extends Action2 {
+	constructor() {
+		super({
+			id: 'editor.action.accessibleViewDisableHint',
+			keybinding: {
+				when: ContextKeyExpr.or(accessibleViewIsShown, accessibilityHelpIsShown),
+				primary: KeyMod.Alt | KeyCode.F6,
+				weight: KeybindingWeight.WorkbenchContrib
+			},
+			icon: Codicon.treeFilterClear,
+			menu: [commandPalette,
+				{
+					id: MenuId.AccessibleView,
+					group: 'navigation',
+					when: ContextKeyExpr.or(accessibleViewIsShown, accessibilityHelpIsShown)
+				}],
+			title: localize('editor.action.accessibleViewDisableHint', "Disable Accessible View Hint")
+		});
+	}
+	run(accessor: ServicesAccessor): void {
+		accessor.get(IAccessibleViewService).disableHint();
+	}
+}
+registerAction2(AccessibleViewDisableHintAction);
+
+class AccessibleViewFocusToolbarAction extends Action2 {
+	constructor() {
+		super({
+			id: 'editor.action.accessibleViewFocusToolbar',
+			keybinding: {
+				when: accessibleViewIsShown,
+				primary: KeyMod.Alt | KeyCode.F7,
```

Comment:
This doesn't seem important enough to have its own keybinding when I'd expect tab or shift-tab to wrap around and focus the actions, does that not happen?

Reply:
yes, that works, I can remove this